lib/reline/unicode.rb

```diff
@@ -193,16 +193,24 @@ def self.split_by_width(str, max_width, encoding = str.encoding)
 
   # Take a chunk of a String cut by width with escape sequences.
   def self.take_range(str, start_col, max_width)
```

Comment:
This method is not used now, but I want to leave it as is because this method might be useful to use it from IRB for string truncation in the future.
If we are going to use it from IRB, we should not release Reline that does not have `Reline::Unicode.take_range`